### PR TITLE
NEPT-1991: Update themes EC Resp, EC Europa and Atomium.

### PR DESCRIPTION
**Pull request by drupol**
### Description

Minor update of the themes.

**drush cc all required.**

### CODE REVIEW

**Comment by drupol** [comment]
Tests are actually not failing: https://platform-ci.ne-dev.eu/jenkins/job/platform-dev/job/nept-1991-release-2.4/5/consoleFull

The issue comes from the communication between Jenkins and Github: 
```
Could not update commit status. Message: Server returned HTTP response code: 201, message: 'Created' for URL: https://api.github.com/repos/ec-europa/platform-dev/statuses/6ee38dc9eb53db8583752a7c8cfe22ac46e14b31

ERROR: script returned exit code 143
Finished: FAILURE
```

File: resources/multisite_drupal_standard.make [hunk 1]
```diff
@@ -1042,12 +1042,12 @@ libraries[respond][download][url] = https://raw.githubusercontent.com/scottjehl/
 projects[ec_resp][type] = theme
 projects[ec_resp][download][type] = git
 projects[ec_resp][download][url] = https://github.com/ec-europa/ec_resp.git
-projects[ec_resp][download][tag] = 2.3.8
+projects[ec_resp][download][tag] = 2.3.9
 
 projects[atomium][type] = theme
-projects[atomium][version] = 2.8
+projects[atomium][version] = 2.10
```
Comment (voidtek) [inline resources/multisite_drupal_standard.make]: Should not we use the version 2.11?
https://github.com/ec-europa/atomium/releases/tag/7.x-2.11 ?

File: resources/multisite_drupal_standard.make [hunk 2]
```diff
@@ -1042,12 +1042,12 @@ libraries[respond][download][url] = https://raw.githubusercontent.com/scottjehl/
...
 
 projects[ec_europa][type] = theme
 projects[ec_europa][download][type] = git
 projects[ec_europa][download][url] = https://github.com/ec-europa/ec_europa.git
-projects[ec_europa][download][tag] = 0.0.11
+projects[ec_europa][download][tag] = 0.0.12
```
Comment (voidtek) [inline resources/multisite_drupal_standard.make]: Should not we use the version 0.0.13?
https://github.com/ec-europa/ec_europa/releases/tag/0.0.13

**Comment by drupol** [comment]
This pr can be closed. See #2092

File: resources/multisite_drupal_standard.make [hunk 3]
```diff
@@ -1042,12 +1042,12 @@ libraries[respond][download][url] = https://raw.githubusercontent.com/scottjehl/
 projects[ec_resp][type] = theme
 projects[ec_resp][download][type] = git
 projects[ec_resp][download][url] = https://github.com/ec-europa/ec_resp.git
-projects[ec_resp][download][tag] = 2.3.8
+projects[ec_resp][download][tag] = 2.3.9
```
Comment (voidtek) [inline resources/multisite_drupal_standard.make]: The version 2.4 is blocked and only for bug-fixing. It's not necessary to upgrade ec_resp theme.

Comment (drupol) [reply resources/multisite_drupal_standard.make]: Ha ????

**Pull request closed without merging**